static/app/components/gridEditable/index.tsx

```diff
@@ -314,7 +314,9 @@ class GridEditable<
         {prependColumns &&
           columnOrder?.length > 0 &&
           prependColumns.map((item, i) => (
-            <GridHeadCellStatic key={`prepend-${i}`}>{item}</GridHeadCellStatic>
+            <GridHeadCellStatic data-test-id="grid-head-cell-static" key={`prepend-${i}`}>
```

Comment:
I would have preferred to know what was supposed to be rendered here instead of using `data-test-id` but since I did not have an example and I am not very familiar with this code, `data-test-id` helped me. 

Plant to revisit this afterward

Reply:
Are you using screen.debug? That’s how I’ve been doing all my conversions. Render and do a screen debug and see what to click on

Reply:
Pretty sure at this point my brain is half WebKit interpreter, since I just look at the html lol

Reply:
yes I debugged the test but the point is that this is not suppose to render, so there is nothing to check :)